src/main/java/eu/seal/apigw/cl/rest_api/services/identsource/ClModuleIDRetrieveGetServiceImp.java

```diff
@@ -70,12 +70,13 @@ public ModuleTrigger clModuleIDRetrieveGet (String sessionID, String moduleID) t
 			switch (moduleID.toLowerCase()) {
 			
 				case "uport":
+				case "sovrin":
```

Comment:
Wow I didn't know we were also using Sovrin

Reply:
Only uPort till the moment :-)